src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="page__month month month--month-length--31">
+      <div class="month__day month__day--start-day--sun"></div>
```

Comment:
start-day modificator should be used in your container class = page__month, not in your day

Reply:
done